lib/code_corps/model/project_user.ex

```diff
@@ -55,7 +54,11 @@ defmodule CodeCorps.ProjectUser do
     |> validate_inclusion(:role, roles())
   end
 
-  defp roles do
-    ~w{ pending contributor admin owner }
+  @doc ~S"""
+  Returns supported project user role types
+  """
+  @spec roles :: list(String.t())
+  def roles do
+    ~w{pending contributor admin owner}
```

Comment:
Did an auto-formatting of the file as I was making the `roles` function public (to be used when validating a `UserInvite`